.circleci/config.yml

```diff
@@ -13,32 +13,57 @@ commands:
     description: "Prepare tooling, setup path for environment"
     steps:
       - run:
-          name: Prepare codebase
+          name: Prepare tooling
           command: |
+            # Prepare some variables to use in commands.
+            ~/project/.circleci/prepare-environment-variables.sh
+            # Install Git and uzip
             apt update && apt install git -y
+            apt install unzip -y
+            # Install Mariadb Client for using mysql command like mysql, drush sql-cli ...
+            apt install mariadb-client -y
+            # Install Platformsh CLI to interact with Platformsh.
+            docker-php-ext-install pcntl
+            docker-php-ext-install posix
+            curl -sS https://platform.sh/cli/installer | php
+            echo "export PATH=$PATH:$HOME/.platformsh/bin" >> $BASH_ENV
```

Comment:
Why don't you move these into `prepare-environment-variables.sh`

Reply:
Because purpose of prepare-environment-variables.sh is for defining variables only.
These things is for prepare tooling like: git, platformsh cli, mysql client ..

Reply:
```
echo "export PATH=$PATH:$HOME/.platformsh/bin" >> $BASH_ENV
```
This looks like defining a variable.